src/test/resources/pages/adminEmailCompose.html

```diff
@@ -27,7 +27,7 @@
          </script>
          <script src="/tinymce/js/tinymce/tinymce.min.js" type="text/javascript">
          </script>
-         <link href="http://localhost:8888/tinymce/js/tinymce/skins/lightgray/skin.min.css" rel="stylesheet"/>
```

Comment:
Yup thanks! will be doing that in #4355. Btw, could you open an issue to update "areTestAccountDefaultValues()"? thanks!